src/data/taskList.ts

```diff
@@ -65,21 +77,5 @@ export const bizCaseStatus = (
     default:
       return 'CANNOT_START';
   }
-};
-
-// chooseBusinessCasePath won't return a link if the business case
-// status is NOT_NEEDED or CANNOT_START
-export const chooseBusinessCasePath = (
```

Comment:
I don't think this is necessary. The function is only invoked in one place. See comment in `src/views/GovernanceTaskList/index.tsx`